src/ui/public/agg_types/buckets/terms.js

```diff
@@ -16,12 +16,16 @@ export default function TermsAggDefinition(Private) {
   const createFilter = Private(AggTypesBucketsCreateFilterTermsProvider);
   const routeBasedNotifier = Private(routeBasedNotifierProvider);
 
-  const aggFilter = ['!top_hits', '!percentiles', '!median', '!std_dev', '!derivative'];
+  const aggFilter = [
+    '!top_hits', '!percentiles', '!median', '!std_dev',
+    '!derivative', '!cumulative_sum', '!moving_avg', '!serial_diff'
```

Comment:
completely forgot i put this here. it should go in separate PR ... but yeah, now its here, so i think as long as i update the commit msg it should be ok ?